channels_irc/client.py

```diff
@@ -265,7 +246,18 @@ def reconnect_checker(self):
         disconnected, and re-starts the connection if it has
         """
         if not self.connection.connected and self.autoreconnect:
-            self.connect(*self._saved_connect.args, **self._saved_connect.kwargs)
+            logger.log('Attempting to reconnect..')
```

Comment:
just outta curiosity, is this a default log level (i.e., from `settings`) or does it default to `debug`. The docs were unclear

Reply:
Huh, you know, I changed this from a print to a log without testing, and I actually think this will raise an exception.  I had to dig into the source to answer your question, but LEVEL is actually a required argument for `.log`.  This should probably just be `info` or `debug`.  Nice catch.

I'm working on some updates so that this can handle multiple connections simultaneously (so we can, say, post using NRG's account instead of ours).  I'll include this fix.